src/cmap/wire_protocol/on_demand/document.ts

```diff
@@ -247,7 +254,7 @@ export class OnDemandDocument {
   public get<const T extends keyof JSTypeOf>(
     name: string | number,
     as: T,
-    required?: false | undefined
+    required?: false | undefined | boolean
```

Comment:
Is this typed like this for a particular reason? Why not just `boolean | undefined`?

Reply:
Oversight, fixed!